packages/helpers/package.json

```diff
@@ -23,6 +23,7 @@
   "license": "GPL-3.0",
   "private": false,
   "dependencies": {
+    "@wordpress/element": "^6.14.0",
```

Comment:
Why this version?

Our lowest support WP version is 6.5:
https://github.com/WordPress/wordpress-develop/blob/6.5.0/package.json#L113C3-L113C31